### PR TITLE
ETFE-2081 Adding CAM-TA02

### PR DESCRIPTION
**Pull request by grahampaulcook**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -311,3 +311,10 @@ transportArranger.4 = Other
 transportArranger.checkYourAnswersLabel = transportArranger
 transportArranger.error.required = Select who is arranging the transport for this movement
 transportArranger.change.hidden = TransportArranger
+
+transportArrangerName.3.title = What is the goods owner''s business name?
+transportArrangerName.4.title = What is the consignee''s business name?
```
Comment (stephenkward) [inline conf/messages.en:316]: This is wrong.  It should be **What is the transport arranger's business name?**

File: conf/messages.en [hunk 2]
```diff
@@ -311,3 +311,10 @@ transportArranger.4 = Other
...
+transportArrangerName.3.heading = What is the goods owner''s business name?
+transportArrangerName.4.heading = What is the consignee''s business name?
```
Comment (stephenkward) [inline conf/messages.en:318]: This is wrong.  It should be **What is the transport arranger's business name?**

File: conf/messages.cy [hunk 3]
```diff
@@ -251,3 +251,11 @@ transportArranger.4 = Other
 transportArranger.checkYourAnswersLabel = transportArranger
 transportArranger.error.required = Select who is arranging the transport for this movement
 transportArranger.change.hidden = TransportArranger
+
+transportArrangerName.3.title = What is the goods owner''s business name?
+transportArrangerName.4.title = What is the consignee''s business name?
```
Comment (stephenkward) [inline conf/messages.cy:256]: This is wrong.  It should be **What is the transport arranger's business name?**

File: conf/messages.cy [hunk 4]
```diff
@@ -251,3 +251,11 @@ transportArranger.4 = Other
...
+transportArrangerName.3.heading = What is the goods owner''s business name?
+transportArrangerName.4.heading = What is the consignee''s business name?
```
Comment (stephenkward) [inline conf/messages.cy:258]: This is wrong.  It should be **What is the transport arranger's business name?**

File: test-utils/fixtures/messages/sections/transportArranger/TransportArrangerNameMessages.scala [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+  sealed trait ViewMessages extends BaseMessages { _: i18n =>
+    def title()(implicit transportArranger: TransportArranger) = transportArranger match {
+      case GoodsOwner => titleHelper("What is the goods owner's business name?")
+      case Other => titleHelper("What is the consignee's business name?")
+    }
+
+    def heading()(implicit transportArranger: TransportArranger) = transportArranger match {
+      case GoodsOwner => "What is the goods owner's business name?"
+      case Other => "What is the consignee's business name?"
```
Comment (stephenkward) [inline test-utils/fixtures/messages/sections/transportArranger/TransportArrangerNameMessages.scala:33]: This is wrong. It should be What is the transport arranger's business name?

File: test-utils/fixtures/messages/sections/transportArranger/TransportArrangerNameMessages.scala [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+import fixtures.messages.{BaseEnglish, BaseMessages, BaseWelsh, i18n}
+import models.sections.transportArranger.TransportArranger
+import models.sections.transportArranger.TransportArranger.{GoodsOwner, Other}
+
+object TransportArrangerNameMessages {
+
+  sealed trait ViewMessages extends BaseMessages { _: i18n =>
+    def title()(implicit transportArranger: TransportArranger) = transportArranger match {
+      case GoodsOwner => titleHelper("What is the goods owner's business name?")
+      case Other => titleHelper("What is the consignee's business name?")
```
Comment (stephenkward) [inline test-utils/fixtures/messages/sections/transportArranger/TransportArrangerNameMessages.scala:28]: This is wrong. It should be What is the transport arranger's business name?

**Review by stephenkward — CHANGES_REQUESTED** [review]

File: conf/messages.en (hunk 1 above)
Comment (grahampaulcook) [reply conf/messages.en]: Fixed on https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/45/commits/3d8b6d565c86fd8cd96af9b999c26311ffb59e71

File: conf/messages.en (hunk 2 above)
Comment (grahampaulcook) [reply conf/messages.en]: Fixed on https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/45/commits/3d8b6d565c86fd8cd96af9b999c26311ffb59e71

File: conf/messages.cy (hunk 3 above)
Comment (grahampaulcook) [reply conf/messages.cy]: Fixed on https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/45/commits/3d8b6d565c86fd8cd96af9b999c26311ffb59e71

File: conf/messages.cy (hunk 4 above)
Comment (grahampaulcook) [reply conf/messages.cy]: Fixed on https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/45/commits/3d8b6d565c86fd8cd96af9b999c26311ffb59e71

File: test-utils/fixtures/messages/sections/transportArranger/TransportArrangerNameMessages.scala (hunk 5 above)
Comment (grahampaulcook) [reply test-utils/fixtures/messages/sections/transportArranger/TransportArrangerNameMessages.scala]: Fixed on https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/45/commits/3d8b6d565c86fd8cd96af9b999c26311ffb59e71

File: test-utils/fixtures/messages/sections/transportArranger/TransportArrangerNameMessages.scala (hunk 6 above)
Comment (grahampaulcook) [reply test-utils/fixtures/messages/sections/transportArranger/TransportArrangerNameMessages.scala]: Fixed on https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/45/commits/3d8b6d565c86fd8cd96af9b999c26311ffb59e71

**Review by stephenkward — APPROVED** [review]

**Pull request merged**